get.sh

```diff
@@ -250,7 +250,15 @@ downloadFile() {
 installFile() {
     tar xf "$TMP_FILE" -C "$TMP_ROOT"
     echo "Preparing to install $BINARY_NAME ${TAG_WO_VER} into ${BIN_INSTALL_DIR}"
-    runAsRoot cp "$TMP_ROOT/$BINARY_NAME" "$BIN_INSTALL_DIR/$BINARY_NAME"
+    # If SUID setup is running for the first time
+    if [ ! -f /etc/containerlab/suid_setup_done ]; then
+        # Set SUID bit on the containerlab binary
+        runAsRoot install -m 4755 "$TMP_ROOT/$BINARY_NAME" "$BIN_INSTALL_DIR/$BINARY_NAME"
+        # Add clab admins group and add current user to it
+        runAsRoot groupadd -r clab_admins
+        runAsRoot usermod -aG "$SUDO_USER" clab_admins
+        runAsRoot touch /etc/containerlab/suid_setup_done
```

Comment:
same comment as in https://github.com/srl-labs/containerlab/pull/2408/files#r1929715066